js/pages/samples.js

```diff
@@ -459,19 +459,45 @@ const u_chicago = {
-        ]
+            '<span id="uln">UChicago, Duchossois Center for Advanced Medicine (DCAM) in Hyde Park</span>',
+            'University of Chicago Medicine <br>Duchossois Center for Advanced Medicine (DCAM) #2101 </br>5758 S. Maryland Avenue <br>Chicago, IL 60637</br><br>After entering the DCAM building from the main entrance, look for us at the top of the stairs on the 2nd floor.</br></p>',
+            '',
+            'The University of Chicago Medicine offers valet and self-parking. We will validate your parking pass. Please show your self-parking ticket to research staff.',
+        ],
+        [
+            '<span id="uln">UChicago Ingalls Memorial Hospital in Harvey, IL</span>',
+            'Ingalls Outpatient Center, Suite #212</br>71 W. 156th St.<br>Harvey, IL 60426</br>',
+            '',
+            'Ingalls Memorial Hospital offers free valet and self-parking.',
+        ],
+        [
+            '<span id="uln">UChicago Medicine – South Loop, in Downtown Chicago</span>',
+            'Southgate Market P1 <br>1101 S Canal St </br> Chicago, IL 60607',
+            '',
+            'The clinic is located on the P1 level next to the DSW. Entrances are located off of Canal St right beside the Panera Bread and on the P1 level of the parking garage. We will validate your parking pass from the parking garage. Please show your self-parking ticket to research staff. We are unable to validate street parking.',
+        ],
+        [
+            '<span id="uln">UChicago Medicine – River East, in Downtown Chicago</span>',
```

Comment:
Change HTML `id` attribute to `class` attribute and provide a descriptive class name value.